torch/quantization/__init__.py

```diff
@@ -44,6 +45,7 @@ def default_eval_fn(model, calib_data):
     # Observers
     'ObserverBase', 'WeightObserver', 'observer', 'default_observer',
     'default_weight_observer', 'default_placeholder_observer',
+    'Logger',
```

Comment:
should this be removed, since the check was removed?

Reply:
oh, yeah will do. looks like CI is passing